ambariclient/base.py

```diff
@@ -590,9 +590,11 @@ def url(self):
         Otherwise, it will generated it based on the collection's url and the
         model's identifier.
         """
-        if self._href is not None:
-            return self._href
+        # if self._href is not None:
+        #     return self._href
```

Comment:
Is there a reason to keep this code here commented out? Otherwise, good to merge.